code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```
@@ -822,7 +822,8 @@ func (fsh *StorageHandler) DeleteFile(ctx context.Context, r *http.Request, conn
 		return nil, common.NewError("invalid_parameters", "Invalid path")
 	}
 
-	fileRef, _ := reference.GetReference(ctx, connectionObj.AllocationID, path)
+	//fileRef, _ := reference.GetReference(ctx, connectionObj.AllocationID, path)
+	fileRef, _ := reference.GetReferenceDelete(ctx, connectionObj.AllocationID, path)
```

@abouroumine Please don't ignore errors. Atleast log it.